libgrabsite/main.py

```diff
@@ -62,6 +62,9 @@ def print_version(ctx, param, value):
 @click.option('--page-requisites-level', default="5", metavar='NUM',
 	help='Recursive this many levels for page requisites (default: 5).')
 
+@click.option('--warc-max-size', default="5368709120", metavar='NUM',
+	help='If a WARC file has grown to this size a new WARC file is created (default: 5368709120).')
```

Comment:
"Try to limit each WARC file to around BYTES bytes before rolling over to a new WARC file."
also describe that that number is 5GB

(I say "try" because a huge response may send it *way* over the limit.)